.changeset/brown-pants-press.md

```diff
@@ -0,0 +1,6 @@
+---
+"@rocket.chat/meteor": major
+"@rocket.chat/i18n": major
+---
+
+Changed some displays to reflect new Private Apps rules and added a new modal before uploading a private app
```

Comment:
According to https://developer.rocket.chat/docs/development-workflow#adding-changeset-to-your-pull-request
> Verb: Start with a verb in the third-person singular present tense.